schemas/2016-06-01/Microsoft.RecoveryServices.json

```diff
@@ -85,40 +82,25 @@
             {
               "type": "string",
               "enum": [
+                "Standard",
                 "RS0"
               ]
             },
             {
               "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
             }
           ],
-          "description": "Gets or sets the sku name. Required for vault creation, optional for update. Possible values include: 'RS0'"
-        },
-        "tier": {
-          "oneOf": [
-            {
-              "type": "string",
-              "enum": [
-                "Standard"
-              ]
-            },
-            {
-              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
-            }
-          ],
-          "description": "Gets or sets the sku tier. Required for vault creation, optional for update. Possible values include: 'Standard'"
+          "description": "The Sku name."
         }
       },
       "required": [
-        "name",
-        "tier"
+        "name"
       ],
-      "description": "The SKU of the recovery services vault."
+      "description": "Identifies the unique system identifier for each Azure resource."
     },
-    "RecoveryServicesPropertiesCreateParameters": {
+    "VaultProperties": {
       "type": "object",
-      "properties": {
```

Comment:
Vault creation requires empty properties bag. will removing this work without any issue?